src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,81 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private final RegistrationService registrationService = new RegistrationServiceImpl();
+    private final StorageDao storageDao = new StorageDaoImpl();
+    private final User defaultUser = new User();
+    private final String correctLogin = "Johnson";
+    private final int correctAge = 20;
+    private final String correctPassword = "John328";
+
+    @Test
+    void correctData_ok() {
+        defaultUser.setLogin(correctLogin);
+        defaultUser.setAge(correctAge);
+        defaultUser.setPassword(correctPassword);
+        storageDao.add(defaultUser);
+        boolean actual = defaultUser.equals(registrationService.register(defaultUser));
+        assertTrue(actual);
+    }
```

Comment:
I think we should check what data is in our db. In this case db must have one element(defaultUser)